LBHackney-IT/lbh-social-care
# conditional render of steps

**What**  
- Added conditional render of steps

Every step can have a `conditionalRender` property where it's possible to do some checks on the `formData`:
```
conditionalRender: ({ conditional_trigger }) =>
        conditional_trigger === 'Yes',
```
If the condition is `true` the step renders, otherwise it jumps to the next one,


- Improved Breadcrumbs component
- Added some test coverage to some crucial parts of the `FormWizard`

The breadcrumbs are aligned to https://design-system.service.gov.uk/components/breadcrumbs/ and now it's only possible to navigate back to steps already completed.
<img width="312" alt="Screenshot 2020-11-21 at 15 25 38" src="https://user-images.githubusercontent.com/435399/99881928-1866dc00-2c1d-11eb-8239-303287e2bb65.png">


**How to test it**
I created a test branch `feature/multi-step-test` that is including a test form.
- go to `/form/multi-step/step-a`
- select "yes"
- continue filling the form
- on summary check that everything looks ok
- go back to the first step
- select "no"
- check that the value initially added is not present in the summary and it's not submitted
